Prefer MIME type over @type for download/display URLs (#7816)
plone/volto: docs/source/configuration/settings-reference.md

```
````

### `downloadableObjects`

This setting defines which content types or MIME types should be treated as downloadable. 
When a content type or MIME type matches an entry in this list, the system will download it instead of displaying it inline.

**Default:**

```js
config.settings.downloadableObjects = ['File'];
```

**Example:**

```js
// If you have a custom content type (e.g. "MyCustomContentType") 
// that stores files, you can list it here to make those files downloadable.
config.settings.downloadableObjects = ['File', 'MyCustomContentType'];

// or
config.settings.downloadableObjects.append('MyCustomContentType')
```

**Notes:**

- The MIME type takes precedence over content type if present and recognized.

> - Objects not listed here but included in `viewableInBrowserObjects` will be displayed inline in the browser.

Reviewer: ```suggestion
- Types (content type and MIME type) included in `viewableInBrowserObjects` will be displayed inline in the browser.
  Setting `viewableInBrowserObjects` overrides `downloadableObjects`.
```

```
````

### `downloadableObjects`

This setting defines which content types or MIME types should be treated as downloadable instead of going to the default view when a logged-out user clicks a link to the item.
When a content type or MIME type matches an entry in this list, the system will download it using the `@@download` view.

> **Default:**

Reviewer 2: Please don't use bold text for these. It's unnecessary and shout-y, almost as bad as ALLCAPS.

```suggestion
Default setting:
```

```js
config.settings.downloadableObjects = ['File'];
```

**Example:**

```js
// If you have a custom content type (e.g. "MyCustomContentType") 

> // that stores files, you can list it here to make those files downloadable.

Reviewer 2: ```suggestion
```

config.settings.downloadableObjects = ['File', 'MyCustomContentType'];

> // or

Reviewer 2: ````suggestion
```

Alternatively, you can append it to the list.

```js
````

config.settings.downloadableObjects.append('MyCustomContentType')
```

**Notes:**

- The MIME type takes precedence over content type if present and recognized.
- Types (content type and MIME type) included in `viewableInBrowserObjects` will be displayed inline in the browser.

>   Setting `viewableInBrowserObjects` overrides `downloadableObjects`.

Reviewer 2: Use an admonition.

````suggestion
```{note}
-   The MIME type takes precedence over content type if present and recognized.
-   Types (content type and MIME type) included in `viewableInBrowserObjects` will be displayed inline in the browser.
    Setting `viewableInBrowserObjects` overrides `downloadableObjects`.
```
````

```{seealso}

> The `viewableInBrowserObjects` setting to configure which content/mime-types should be opened inline in the browser.

Reviewer 2: ```suggestion
The :ref:`viewableInBrowserObjects` setting to configure which content or MIME types should be opened inline in the browser.
```

`unwantedControlPanelsFields` is the control panel fields that are not used in Volto.
It is used internally by the `filterControlPanelsSchema` function.

> ### `viewableInBrowserObjects`

Reviewer 2: Add MyST label for the above reference link.

```suggestion
(viewableInBrowserObjects)=

### `viewableInBrowserObjects`
```

This setting defines which content types or MIME types should be displayed inline in the browser instead of going to the default view when a logged-out user clicks a link to the item.

> When an content type or MIME type matches an entry in this list, the system will open it inline in a new browser tab using the `@@display-file` view.

Reviewer 2: ```suggestion
When a content type or MIME type matches an entry in this list, the system will open it inline in a new browser tab using the `@@display-file` view.
```

> **Default:**

Reviewer 2: ```suggestion
Default setting:
```

```js
config.settings.viewableInBrowserObjects = [];
```

> **Example:**

Reviewer 2: ```suggestion
The following example will display only PDFs inline.
```

```js

> // Only PDFs will be viewable inline

Reviewer 2: ```suggestion
```

config.settings.viewableInBrowserObjects = ['application/pdf'];
```

**Notes:**

- The MIME type takes precedence over content type if present and recognized.

> - Objects not listed here but included in `downloadableObjects` will be downloaded instead.

Reviewer 2: ````suggestion
```{note}
-   The MIME type takes precedence over content type if present and recognized.
-   Objects not listed here but included in `downloadableObjects` will be downloaded instead.
```
````
